spec/cucumberConf.js

```diff
@@ -0,0 +1,35 @@
+var env = require('./environment');
```

Comment:
You could put something in `docs/`, just make sure it makes clear we don't provide first-party support